flows/aws/interactive-workflow/README.md

```diff
@@ -45,7 +46,11 @@ pip install -r requirements.txt
 
   Make sure Prefect is running and you are logged into Prefect Cloud or your local Prefect instance.
 
-2. **Execute the Workflow**
+2. **Prefect & Marvin environment is set up**
+
+Ensure your S3Bucket block is correctly set up to upload files to that specific region. Additionally, ensure you have the OpenAI api key correctly loaded into Marvin. Once you are authenticated to the correct workspace, you are good to go to run the workflow.
```

Comment:
It would be helpful to either link to the `S3Bucket` and Marvin docs, or mention the important bits here.